.travis.yml

```diff
@@ -28,23 +26,34 @@ jobs:
 
   - name: Backend E2E Tests - Openshift
     script:
+    - travis_retry scripts/setup-tools-for-ubuntu.sh
     - travis_retry scripts/setup-openshift.sh ${OC_TOKEN}
     - make -C enterprise-suite openshift-backend-tests
 
   - name: Backend E2E Go Tests - Minikube
+    before_script:
+    - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin ${DOCKER_REGISTRY}
+    - travis_retry docker pull ${DOCKER_REGISTRY}/${GOTESTS_CONTAINER}
```

Comment:
Can we call it BUILD_CONTAINER (idea being we can use it for the frontend tests as well, it's not gotests specific).